### PR TITLE
removes DigitalOcean from INTHEWILD.md

### PR DESCRIPTION
**Pull request by nmyk**
Airflow is no longer used at DigitalOcean.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by turbaszek — APPROVED** [review]

**Comment by turbaszek** [comment]
If I can ask out of curiosity, why you stoped using Airflow?

**Comment by nmyk** [comment]
Sure! We had a key team member leave the company a few years ago who was the resident Airflow expert. Around that same time, we were starting work on a revamp of our entire business intelligence stack from the ground up, and figured we might as well reconsider how we schedule jobs while we were at it. We ended up designing a new system of data pipelines that really only requires a simple time-based scheduler like `cron` as we found other ways of handling dependencies etc.

tl;dr It's not you, it's us 😆

**Comment by turbaszek** [comment]
Thanks a lot @nmyk !
